src/commands/scanner/rule/list.ts

```diff
@@ -68,6 +68,12 @@ export default class List extends ScannerCommand {
 			description: messages.getMessage('flags.customDescription'),
 			exclusive: ['standard'],
 			hidden: true
+		}),
+		engine:  flags.array({
```

Comment:
There should probably be a comment above this new flag indicating that it's relevant to the FilterTypes stuff defined in `ScannerCommand`. It also wouldn't hurt to add that comment above the relevant existing flags too.